umake/frameworks/games.py

```diff
@@ -210,3 +210,42 @@ def post_install(self):
                         exec='"{}" %f'.format(self.exec_path),
                         comment=self.description,
                         categories="Development;IDE;"))
+
+
+class Superpowers(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, category):
+        super().__init__(name="Superpowers", description=_("The HTML5 2D+3D game maker"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         download_page="https://github.com/superpowers/superpowers/releases/latest",
```

Comment:
The github.com/.../.../releases/latest pages in github are indeed a redirect to the tagged one.
@didrocks, is the option needed the "-redirect" in tests/tools/local_server.py?